packages/admin/admin/src/common/toolbar/actions/helpDialogAction/HelpDialogAction.tsx

```diff
@@ -0,0 +1,72 @@
+import { QuestionMark } from "@comet/admin-icons";
+import { type ComponentsOverrides, type DialogContent, type IconButton, type Theme, useThemeProps } from "@mui/material";
+import { type FunctionComponent, type ReactNode, useState } from "react";
+import { FormattedMessage } from "react-intl";
+
+import { type ThemedComponentBaseProps } from "../../../../helpers/ThemedComponentBaseProps";
+import { type Dialog } from "../../../Dialog";
+import { HelpButton, HelpDialog, HelpDialogContent, Root } from "./HelpDialogAction.sc";
+
+export type HelpDialogActionClassKey = "root" | "button" | "dialog" | "dialogContent";
+
+export type HelpDialogActionProps = ThemedComponentBaseProps<{
+    root: "div";
+    button: typeof IconButton;
+    dialog: typeof Dialog;
+    dialogContent: typeof DialogContent;
+}> & {
+    dialogTitle?: ReactNode;
+    dialogDescription: ReactNode;
+    icon?: ReactNode;
```

Comment:
I find it odd that the dialog props are prefixed with "dialog" but the button prop isn't prefix with "button". Should we add a prefix for both or remove the prefix?

Reply:
IMO the prefix is now justified since the other props are the button's props.